core/test/parsers/CMakeLists.txt

```diff
@@ -25,4 +26,4 @@ target_include_directories(ledger-core-parser-tests PUBLIC ../../../qt-host)
 include(CopyAndInstallImportedTargets)
 copy_install_imported_targets(ledger-core-parser-tests crypto)
 
-add_test (NAME ledger-core-parser-tests COMMAND ledger-core-parser-tests)
\ No newline at end of file
+add_test (NAME ledger-core-parser-tests COMMAND ledger-core-parser-tests)
```

Comment:
Could you, please, split this on two separate tests.

Reply:
@KhalilBellakrid merged already, but this wasn’t something I added, I guess it’s a newline conversion. :)